docs/guides/text-tracks.md

```diff
@@ -1,6 +1,6 @@
 # Text Tracks
 
-Text tracks are a feature of HTML5 video for displaying time-triggered text to the viewer. Video.js offers a cross-browser implementation of text tracks.
+Text tracks are a feature of HTML5 for displaying time-triggered text to the user. Video.js offers a cross-browser implementation of text tracks.
```

Comment:
makes to remove `video` there, viewer does imply video but I'm not sure how well user works. Maybe end-user? Since user often implies the user of Video.js, aka, the developer.